docs/nada-by-example.md

```diff
@@ -0,0 +1,13 @@
+---
+displayed_sidebar: nadaByExampleSidebar
+---
+
+# Nada by Example
+
+Welcome to **Nada by Example**, an introduction to the Nada language with reference examples from the [nada-by-example](https://github.com/NillionNetwork/nada-by-example) repo. Nada is a Python-based language for defining MPC (Multi-Party Compute) programs on the [Nillion Network](/network). 
+
+Developers can use Nada to write programs that handle secret inputs from multiple parties (i.e., multiple users). These programs compute in a way that ensures they are “blind” to the underlying data, meaning they cannot access or view the secret inputs. This concept is known as “blind computation.”
+
+To learn more about Nada, check out the full [Nada language docs here](/nada-lang).
+
+Version: [nada-dsl 0.4.0](https://pypi.org/project/nada-dsl/0.4.0/)
```

Comment:
Wouldn't it make more sense to point to a generic version of the nada-dsl? I think we're releasing 0.5.0 today, so this would become outdated.

Reply:
great point - I removed the version to just point to nada-dsl